notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.        mo_command = f"""mo
</pre><p>model = mo.convert_model(...)</p><p>BTW, looks like you subtract mean_values twice, this parameter should be removed. It is equivalent of mean_values in preprocessing api</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I think we also should add some imention about mean/scale values and reverse_input_channels using mo as alternative and avoiding double work like in your case

Reply:
add description and change to mo.convert_model